packages/components/src/index.js

```diff
@@ -68,7 +68,7 @@ import {
 
 import I18N_DOMAIN_COMPONENTS from './constants';
 
-import { Action, ActionDropdown, ActionSplitDropdown, Actions } from './Actions';
+import { Action, ActionDropdown, ActionButton, ActionSplitDropdown, Actions } from './Actions';
```

Comment:
should ActionDropdown, ActionButton, ActionSplitDropdown  still be exported ? Since Action will render to those given the right props ?

Reply:
if no dependencies on this and anyone use it alone